client/layout/guided-tours/tours/gdocs-integration-tour.js

```diff
@@ -16,6 +16,11 @@ import {
 	Quit,
 } from 'layout/guided-tours/config-elements';
 import { hasUserPastedFromGoogleDocs } from 'state/ui/guided-tours/contexts';
+import analytics from 'lib/analytics';
+
+const trackUserInterest = ( ) => {
```

Comment:
please remove this whitespace

> No filler spaces in empty constructs (e.g., `{}`, `[]`, `fn()`).

from https://wpcalypso.wordpress.com/devdocs/docs/coding-guidelines/javascript.md?term=No%20filler%20spaces%20in%20empty%20constructs